quickwit/Cargo.lock

```diff
@@ -78,6 +78,12 @@ version = "0.1.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "250f629c0161ad8107cf89319e990051fae62832fd343083bea452d93e2205fd"
 
+[[package]]
+name = "android-tzdata"
```

Comment:
if someone is wondering this is not a real dep. It only is enabled when targetting android.